pkg/cmd/pulumi/package_publish.go

```diff
@@ -141,13 +141,13 @@ func publishToNPM(path string) error {
 	infoCmd := exec.Command(npm, "info", pkgNameWithVersion)
 	infoCmd.Stderr = os.Stderr
 	logging.V(1).Infof("Running %s", infoCmd)
-	output, err := infoCmd.Output()
-	if err != nil {
-		return err
-	}
+	// we actually do not care about the error here; we care whether the output is empty.
+	output, _ := infoCmd.Output()
 
 	if len(output) > 0 {
-		return fmt.Errorf("publish %s failed, likely because the version already exists", pkgInfo.Name)
+		// the package already exists, and we no-op.
+		fmt.Printf("did not publish %s, likely because the version %s already exists\n", pkgInfo.Name, pkgNameWithVersion)
```

Comment:
Is "likely" actually the right word here? At this point isn't it definite that "npm info" only returns data because this specific version is already published?